immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>I'll go back to this consideration, after refactoring the pipeline lab 02, so that the Vertex training &amp; deployment code is as similar as possible in both labs. </p>

Reply:
<p>If you prefer to first merge this in as it is and later unify the interface, sgtm too.</p>

Reply:
<p>Let's do that and revisit in a last path on all the 3 labs for coherence. </p>